Resources/Prototypes/Entities/Objects/Materials/materials.yml

```diff
@@ -479,6 +486,8 @@
     tags:
       - ClothMade
       - RawMaterial
+  - type: Item
+    heldPrefix: pyrotten
```

Comment:
```suggestion
    heldPrefix: pyrotton
```

Reply:
wait what i thought it was an e not an o lol